ios/stockfish.podspec

```diff
@@ -19,23 +19,32 @@ Pod::Spec.new do |s|
   s.public_header_files = 'Classes/**/*.h'
   s.exclude_files = 'Stockfish/src/incbin/UNLICENCE'
   s.dependency 'Flutter'
-  s.platform = :ios, '11.0'
-  s.ios.deployment_target  = '11.0'
+  s.platform = :ios, '12.0'
+  s.ios.deployment_target  = '12.0'
```

Comment:
Do we require `IPHONEOS_DEPLOYMENT_TARGET>=12` from now on?

Reply:
I just followed Flutter which now supports 12+

Reply:
Let's bump iOS version in README then.

Reply:
👍  I just updated the README